### PR TITLE
Mice are MICE and cats like MICE

### PR DESCRIPTION
**Pull request by Sniblet**
# Document the changes in your pull request

Adds the MICE food group and exhaustively sets all rodent-based foods I know of to be MICE where they would be RAW, MEAT, and/or GROSS.
Cats now like MICE. Other species are not affected by this, as MICE has been accordingly added to their already existing likes and dislikes, on the same line as whatever they like least out of RAW, MEAT or GROSS.
This is what it takes.

- [x] Tested
- [x] Mice are MICE
- [x] Cats like MICE
- [x] Works

I'm going to keep proposing changes for felinids to be more and more like real cats until someone just gives them the tajaran sprite and then I'll request a community ban

# Wiki Documentation

Felinids now like mice

# Changelog
:cl:  
tweak: Felinids now like mice
/:cl:

### CODE REVIEW

**Comment by tattax** [comment]
changelog is too much

**Comment by ToasterBiome** [comment]
too op

**Comment by Chubbygummibear** [comment]
more overlap with lizards, felinid/lizard convergence event when?

**Comment by Sniblet** [comment]
> too op

if this is a real concern I can move DAIRY to toxic because lactose intolerance is a thing cats have

**Comment by ghost** [comment]
Pay to win

**Comment by Marmio64** [comment]
Make felinids start with a phobia of snake plushies

**Comment by ghost** [comment]
Make felenids burn and take damage from holy water

**Review by ToasterBiome — APPROVED** [review]

**Comment by ToasterBiome** [comment]
I personally think this is op

**Comment by Sniblet** [comment]
Deviant tastes is free

**Comment by ghost** [comment]
> I personally think this is op

Yes it is.

**Comment by Sniblet** [comment]
TL;DR
check edit history if you still care to see me rant in defense of this after I've changed how it works

**Comment by Sniblet** [comment]
after testing this doesn't even make cats like mice which was the main intent
putting on draft while i figure out what went wrong this time

**Comment by Sniblet** [comment]
Alright turns out taste code is weird and spacemen are incredibly picky, I can't see a way to make cats like mice without either radically changing lizards' palate, radically changing everyone's palate, or allowing cats to suddenly like a lot more things than they have any reason to
You win this one

**Comment by Sniblet** [comment]
FOR NOW

**Comment by Sniblet** [comment]
i have it
editing description to match the change

**Comment by Sniblet** [comment]
WHY DOESN'T THIS WORK I FELT SO CLEVER AAAARRGGGHHH BURN THIS CODE TO THE GROUND
For some reason cats still dislike mice - I guess because they somehow dislike RAW?
I'm gonna leave this PR alone for a bit while I go to bed and test this in private when I wake up
Sorry about the spam, and this is not ready for HC review

**Comment by ghost** [comment]
Heresy

**Comment by Chubbygummibear** [comment]
> Alright turns out taste code is weird and spacemen are incredibly picky, I can't see a way to make cats like mice without either radically changing lizards' palate, radically changing everyone's palate, or allowing cats to suddenly like a lot more things than they have any reason to You win this one

it's because you didn't override the default human dislikes which include raw and gross. adding them to liked won't wipe them from the inherited disliked group. you'd need to include like a
`DISLIKED FOOD = VEGETABLES` or null or none or what have you 
for them to accept raw and gross

**Comment by Sniblet** [comment]
Actually tested and ready now

**Comment by ghost** [comment]
patrick-bateman

**Comment by NCRLouTenant** [comment]
Sniblet, your power frightens me, and I mean that in the most complimentary way possible.
